slave/centos-7/manifest.yml

```diff
@@ -1,6 +1,6 @@
 images:
 - name: buildslave-centos-7-zfs-head
-  base-ami: ami-c7d092f7
+  base-ami: ami-edffe8dd
```

Comment:
Oops. I've seen that you've fixed it already, which is great. Perhaps the docs somewhere could/should be improved?